### PR TITLE
Add files in the main repo

### LINKED ISSUE #4: Add files in the main repo
To add the following files:

- `README` file
- `project_parameters.Config.yaml`
- `figures` folder: contains color palettes and plot theme to be used across the project

### PR DESCRIPTION
**Pull request by AntoniaChroni**
### Purpose/implementation Section

The current PR adds the following files in the main repo:

1. `README` file: with main information about the repo
2. `project_parameters.Config.yaml`: parameters and path files to be used during the test run analysis.
3. `figures` folder: contains color palettes and plot theme to be used across the project

#### What GitHub issue does your pull request address?

Closes #4 

#### Which areas should receive a particularly close look?

Please confirm the following:

- [x] `README` file contains enough documentation for the repo. This will be updated as we go.
- [x] There is a `project_parameters.Config.yaml` file for the `snRNASeq_of_DS_in_Df1_mice` project and you have access to the `data_dir` and `metadata_dir` files.
- [x] There is `figures` folder and if the documentation in the `README` file is clear.
- [x] There are 6 files in `palettes`.
- [x] There are two files in `img`.
- [x] There are two files in `scripts` .

### CODE REVIEW

File: figures/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Create color palettes and plot theme
+
+### Usage
+
+The `create_color_palette_project.R` script is intended to be run via the command line from the top directory of the repository as follows:
```
Comment (sharonfreshour) [inline figures/README.md:5]: For extra clarity, maybe you could provide an example of the top directory path?

File: figures/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+Rscript create_color_palette_project.R
+```
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `create_color_palette_project.R`: define `root_dir`
```
Comment (sharonfreshour) [inline figures/README.md:12]: Again, for extra clarity, maybe expand to say: define `root_dir` for your specific project path

**Comment by AntoniaChroni** [comment]
@sharonfreshour all hidden files are now removed. See 870f65ea7057b9f8c9c8821119fd27d542170329.

File: figures/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Folder content
+
+This folder contains scripts tasked to (1) provide the plot theme to be used across data analysis, and (2) create color palettes for the project.
+
+## Folder structure 
+
+The structure of this folder is as follows:
+
+```
+├── img
```
[the remark this answers is not in the document]
Comment (AntoniaChroni) [reply figures/README.md:23]: @sharonfreshour This has been addressed here edab250c1b5ece65e4f77dfd80f9f896dd178d44.
I also added the missing color palette files.

File: figures/README.md (hunk 1 above)
Comment (AntoniaChroni) [reply figures/README.md:5]: @sharonfreshour The top directory is the `root_dir` for the user. Is this clear or not?

File: figures/README.md (hunk 2 above)
Comment (AntoniaChroni) [reply figures/README.md:12]: The `root_dir` is the directory where the user has cloned the repo.
Could the following example is clear?

Example:
```
/Users/my_username/Documents/sc-rna-seq-snap
```

Comment (sharonfreshour) [reply figures/README.md:12]: For this one, I was actually meaning just putting the directions as "define `root_dir` for your specific project path". But if you want to provide that example, that works I think

File: figures/README.md (hunk 1 above)
Comment (AntoniaChroni) [reply figures/README.md]: This has been now addressed here 5d50679872637b1bafe945e5852306923e164ccf

File: figures/README.md (hunk 2 above)
Comment (AntoniaChroni) [reply figures/README.md]: This has now been addressed here https://github.com/stjudeDNBBinfCore/sc-rna-seq-snap/commit/5d50679872637b1bafe945e5852306923e164ccf

**Pull request merged**